### PR TITLE
[Lists] Pass `Name` parameter

### PR DESCRIPTION
**Pull request by vnbaaij**
Pass the `Name` parameter to the name attribute of the web components. This is needed to enable SSR scenarios

### CODE REVIEW

File: tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-<fluent-slider class="horizontal" min="-1.5" max="7" step="0.5" id="xxx" value="3.5" blazor:onchange="1" blazor:elementreference="xxx"></fluent-slider>
+<fluent-slider class="horizontal" min="-1.5" max="7" step="0.5" id="xxx" value="3,5" blazor:onsliderchange="1" blazor:elementreference="xxx"></fluent-slider>
```
Comment (dvoituron) [inline tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html:2]: Why did you set `3,5` (with a coma)?

File: src/Core/Components/List/FluentListbox.razor [hunk 2]
```diff
@@ -18,6 +18,7 @@
                         size="@Size"
                         multiple="@Multiple"
                         required="@Required"
+                        name=@Name
```
Comment (dvoituron) [inline src/Core/Components/List/FluentListbox.razor:21]: Could you add double quotes?

File: src/Core/Components/List/FluentSelect.razor [hunk 3]
```diff
@@ -15,6 +15,7 @@
                    appearance="@Appearance.ToAttributeValue()"
                    current-value="@Value"
                    required="@Required"
+                   name=@Name
```
Comment (dvoituron) [inline src/Core/Components/List/FluentSelect.razor:18]: Same

File: src/Core/Components/List/FluentCombobox.razor [hunk 4]
```diff
@@ -22,6 +22,7 @@
                      required="@Required"
                      @onchange="@ChangeHandlerAsync"
                      autofocus="@Autofocus"
+                     name=@Name
```
Comment (dvoituron) [inline src/Core/Components/List/FluentCombobox.razor:25]: Could you add double quotes?

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html (hunk 1 above)
Comment (vnbaaij) [reply tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html:2]: I did not change that one

Comment (vnbaaij) [reply tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html:2]: It is the change you made to the value that is causing this test to fail
`"@BindConverter.FormatValue(Value?.ToString())"` => `@FormatValueAsString(Value)"`

Comment (vnbaaij) [reply tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html]: Ok, was in (unexplainable) issue in the verified file...

**Review by dvoituron — APPROVED** [review]

**Pull request merged**